.github/workflows/ci.yml

```diff
@@ -36,6 +36,12 @@ jobs:
         fetch-depth: 0
         lfs: 'false'
 
+    - name: Setup JDK
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'temurin'
+        java-version: '17'
```

Comment:
what does this do, since it is already added to the docker image ?

Reply:
And why is it not needed in android workflow ?

Reply:
Github runners on macOS and Windows uses JDK 8 without this step that configures JDK 17.  
I need JDK 11+ for the test to run Java files directly without a compilation step (it's compiled on the fly)

Reply:
On the Docker image, the package is already JDK 11, so the Android CI does not need to install another version.

Reply:
I could add a condition `if not linux` on the step to rely on JDK 11 on the regular CI, but I think it's fine like that

Reply:
fine as it is.